cmd/minikube/cmd/start.go

```diff
@@ -1029,6 +1029,10 @@ func validateCPUCount(drvName string) {
 
 	}
 
+	if si.CPUs < cpuCount {
+		exitIfNotForced(reason.RsrcInsufficientCores, "Requested cpu count {{.requested_cpus}} is greater than the available cpus of {{.avail_cpus}}", out.V{"requested_cpus": cpuCount, "avail_cpus": si.CPUs})
```

Comment:
before exit lets add an advice message to them if they are using Docker Desktop to increase their cpu in the docker desktop
```
if driver.IsDockerDesktop(drvName) {
			out.Step(style.Empty, `- Ensure your {{.driver_name}} daemon has access to enough CPU/memory resources.`, out.V{"driver_name": driver})
			if runtime.GOOS == "darwin" {
				out.Step(style.Empty, `- Docs https://docs.docker.com/docker-for-mac/#resources`, out.V{"driver_name": driver})
			}
			if runtime.GOOS == "windows" {
				out.String("\n\t")
				out.Step(style.Empty, `- Docs https://docs.docker.com/docker-for-windows/#resources`, out.V{"driver_name": driver})
			}
		}
}
```

also since this logic
and then do the exitIfNotFroced anyways

Reply:
I'll add the changes and re-push sir :)

Reply:
We already have that (Docker Desktop) warning in other places, but they will also have the same problem for this one.

It's not possible to allocate more CPU to the Docker VM, than what is available on the machine. So nothing to do here ?

Reply:
> We already have that (Docker Desktop) warning in other places, but they will also have the same problem for this one.
> 
> It's not possible to allocate more CPU to the Docker VM, than what is available on the machine. So nothing to do here ?

Agree with your point. I just noticed that :)

Reply:
I think there's no issues with @medyagh 's suggestion. As `CachedDaemonInfo` function will return the docker/podman info. Not the host machine's details. It would be worth displaying an advice message like this before exiting. Please correct me if I'm wrong